flink-filesystems/flink-oss-fs-hadoop/src/main/resources/META-INF/NOTICE

```diff
@@ -6,117 +6,24 @@ The Apache Software Foundation (http://www.apache.org/).
-- net.minidev:json-smart:2.3
-- org.apache.avro:avro:1.8.2
-- org.apache.commons:commons-compress:1.20
-- org.apache.commons:commons-configuration2:2.1.1
-- org.apache.commons:commons-lang3:3.3.2
-- org.apache.commons:commons-math3:3.5
-- org.apache.curator:curator-client:2.12.0
-- org.apache.curator:curator-framework:2.12.0
-- org.apache.curator:curator-recipes:2.12.0
-- org.apache.hadoop:hadoop-aliyun:3.1.0
-- org.apache.hadoop:hadoop-annotations:3.1.0
-- org.apache.hadoop:hadoop-auth:3.1.0
-- org.apache.hadoop:hadoop-common:3.1.0
-- org.apache.htrace:htrace-core4:4.1.0-incubating
-- org.apache.httpcomponents:httpclient:4.5.3
-- org.apache.httpcomponents:httpcore:4.4.6
-- org.apache.kerby:kerb-admin:1.0.1
-- org.apache.kerby:kerb-client:1.0.1
-- org.apache.kerby:kerb-common:1.0.1
-- org.apache.kerby:kerb-core:1.0.1
-- org.apache.kerby:kerb-crypto:1.0.1
-- org.apache.kerby:kerb-identity:1.0.1
-- org.apache.kerby:kerb-server:1.0.1
-- org.apache.kerby:kerb-simplekdc:1.0.1
-- org.apache.kerby:kerb-util:1.0.1
-- org.apache.kerby:kerby-asn1:1.0.1
-- org.apache.kerby:kerby-config:1.0.1
-- org.apache.kerby:kerby-pkix:1.0.1
-- org.apache.kerby:kerby-util:1.0.1
-- org.apache.kerby:kerby-xdr:1.0.1
-- org.apache.kerby:token-provider:1.0.1
-- org.apache.zookeeper:zookeeper:3.4.10
-- org.codehaus.jackson:jackson-core-asl:1.9.2
-- org.codehaus.jackson:jackson-jaxrs:1.9.2
-- org.codehaus.jackson:jackson-mapper-asl:1.9.2
-- org.codehaus.jackson:jackson-xc:1.9.2
-- org.codehaus.jettison:jettison:1.1
-- org.eclipse.jetty:jetty-http:9.3.19.v20170502
-- org.eclipse.jetty:jetty-io:9.3.19.v20170502
-- org.eclipse.jetty:jetty-security:9.3.19.v20170502
-- org.eclipse.jetty:jetty-server:9.3.19.v20170502
-- org.eclipse.jetty:jetty-servlet:9.3.19.v20170502
-- org.eclipse.jetty:jetty-util:9.3.19.v20170502
-- org.eclipse.jetty:jetty-webapp:9.3.19.v20170502
-- org.eclipse.jetty:jetty-xml:9.3.19.v20170502
-- org.xerial.snappy:snappy-java:1.1.4
-
-This project bundles the following dependencies under the BSD license.
-See bundled license files for details.
-
-- com.google.protobuf:protobuf-java:2.5.0
-- com.jcraft:jsch:0.1.54
-- com.thoughtworks.paranamer:paranamer:2.7
-- org.codehaus.woodstox:stax2-api:3.1.4
-- org.ow2.asm:asm:5.0.4
+- org.apache.hadoop:hadoop-aliyun:jar:3.1.0
+- com.aliyun.oss:aliyun-sdk-oss:jar:3.4.1
+- org.apache.httpcomponents:httpclient:jar:4.5.3
+- org.apache.httpcomponents:httpcore:jar:4.4.6
+- commons-logging:commons-logging:jar:1.1.3
+- commons-codec:commons-codec:jar:1.10
+- org.codehaus.jettison:jettison:jar:1.1
+- com.aliyun:aliyun-java-sdk-core:jar:3.4.0
+- com.aliyun:aliyun-java-sdk-ram:jar:3.0.0
+- com.aliyun:aliyun-java-sdk-sts:jar:3.0.0
+- com.aliyun:aliyun-java-sdk-ecs:jar:4.2.0
 
 This project bundles the following dependencies under the Common Development and Distribution License (CDDL) 1.0.
 See bundled license files for details.
 
-- javax.ws.rs:jsr311-api:1.1.1
-- javax.xml.stream:stax-api:1.0-2
 - stax:stax-api:1.0.1
```

Comment:
According to https://mvnrepository.com/artifact/stax/stax-api/1.0.1 and the pom this is Apache licensed. Only `javax.xml.stream:stax-api` uses the CDDL license.

Reply:
I see, then this was incorrect in the `NOTICE` file before. I'll update.

Do you know why it was in this section before?

Reply:
My guess it that they assumed that all `stax-api` dependencies have the same license.
While I double-checked this I got confused as well as I found an entry on one [page](https://enterprise.dejacode.com/components/public/StAX+API/1.0-2/#essentials) listing it as CDDL, with only a tiny detail (the version `1.0-2` in the url) indicating that it is actually the `javax.xml.stream` one.